### PR TITLE
Admin redirects

### LINKED ISSUE #26: User Story 45, Admin is redirected from Merchant Dashboard to User profile
```
As an admin user
If I visit a merchant dashboard, but that merchant is a regular user
Then I am redirected to the appropriate user profile page.

eg, if I visit "/admin/merchants/7" but that merchant is a regular user
then I am redirected to "/admin/users/7" and see their user profile page
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #33

child of #79

### PR DESCRIPTION
**Pull request by teresa-m-knowles**
As an admin user
If I visit a merchant dashboard, but that merchant is a regular user
Then I am redirected to the appropriate user profile page.

eg, if I visit "/admin/merchants/7" but that merchant is a regular user
then I am redirected to "/admin/users/7" and see their user profile page

Closes #26

### CODE REVIEW

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -2,5 +2,6 @@ class Admin::MerchantsController < Admin::BaseController
   def show
     @merchant = User.find(params[:id])
     @orders = Order.pending_orders(@merchant)
+    redirect_to admin_user_path(@merchant) if @merchant.registered?
```
Comment (teresa-m-knowles) [inline app/controllers/admin/merchants_controller.rb:5]: I think because my last branch hasn't been merged yet there are a bunch of changes that appear. It's actually just this one line plus the test for it.

File: spec/features/admins/admin_sees_merchant_dashboard_spec.rb [hunk 2]
```diff
@@ -41,6 +41,14 @@
       expect(current_path).to eq(admin_merchant_path(@merchant))
     end
 
+    it 'redirects me to the user profile if that user is not a merchant' do
```
Comment (teresa-m-knowles) [inline spec/features/admins/admin_sees_merchant_dashboard_spec.rb:44]: This is the test for this PR

**Comment by timnallen** [comment]
Nice job!!! 👍

File: app/controllers/admin/merchants_controller.rb (hunk 1 above)
Comment (csvlewis) [reply app/controllers/admin/merchants_controller.rb:5]: Seems like a quick fix. Good work!

**Review by csvlewis — APPROVED** [review]

**Review by allieIsOkay — APPROVED** [review]
Nice job.
